src/constants/globalStyles.js

```diff
@@ -0,0 +1,4 @@
+export const squareSmall = {
+  maxHeight: 20,
+  maxWidth: 20
```

Comment:
what i meant was to extract only the constant value of 20, because if you want to make it bigger or smaller, you must change both numbers to keep it square sized